lib/acts_as_ar_model.rb

```diff
@@ -1,3 +1,5 @@
+require_relative "extensions/descendant_loader.rb"
```

Comment:
I'm concerned that this breaks Rails autoloading, but that concern is probably minor/unwarranted.